using_images/s2i_images/ruby.adoc

```diff
@@ -96,6 +96,12 @@ and ruby gem installation. The application assets are only compiled if
 Asset compilation only happens when the application runs in a production
 environment. Therefore, you can use this variable when assets have already been
 compiled.
+
+|`*PUMA_MIN_THREADS*`, `*PUMA_MAX_THREADS*`
+|Indicates the minimum and maximum threads that will be available in link:https://github.com/puma/puma[Puma]'s thread pool.
```

Comment:
There are no limits. And I think min thread doesn't necessarily need to be lower than max. It is certainly valid that they are equal, that will cause the number of threads to be exactly that number at all times. I suspect that if min > max, the result will be the same. In any case, we only validate that those values are numbers and if they are, we just pass them to Puma and let it handle the rest.